Makefile

```diff
@@ -16,6 +16,14 @@ PKGS ?= v0.5.0-8-gb0d9cd2
 
 SFYRA_CLUSTERCTL_CONFIG ?= $(HOME)/.cluster-api/clusterctl.sfyra.yaml
 
+GO_BUILDFLAGS ?=
+
+WITH_DEBUG ?=
+
+ifeq ($(WITH_DEBUG), true)
```

Comment:
It will not work with `WITH_DEBUG=0` or `WITH_DEBUG=1`. I'm not fond of it at all. It is very tempting to write a boolean parser in `hack/parsebool.(py|sh)` and invoke it there via shell.

Reply:
another option is to check whether `WITH_DEBUG` is not empty, but that might be overkill

Reply:
Check for empty string will enable debug build with `WITH_DEBUG=0`, and that's even worse